Gemfile.lock

```diff
@@ -330,5 +330,8 @@ DEPENDENCIES
   uglifier
   warden
 
+RUBY VERSION
```

Comment:
Yep- 2.3.0 was set in the Gemfile, so maybe this was a result of me adding it to my .ruby-version. Other than that I don't know what changed.